web/publicendpointresource.go

```diff
@@ -291,118 +291,35 @@ func restAddPort(w *rest.ResponseWriter, r *rest.Request, client *node.ControlCl
 	}
 
 	glog.V(0).Infof("Add PORT with %s %#v", r.URL.Path, r)
-	glog.V(0).Infof("request = %#v", request)
-
-	// Validate the port number
-	scrubbedPort := svc.ScrubPortString(request.PortName)
-	portParts := strings.Split(scrubbedPort, ":")
-	if len(portParts) <= 1 {
-		err := fmt.Errorf("Invalid port address. Port address be \":[PORT NUMBER]\" or \"[IP ADDRESS]:[PORT NUMBER]\"")
-		glog.Error(err)
-		restServerError(w, err)
-		return
-	}
-	port, err := strconv.Atoi(portParts[1])
```

Comment:
What about the checks for reserved ports 5000 and 8080? I'm not sure why we added those, but this change looks like it will allows users to add an endpoint-by-port for those values.